ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.17.yaml

```diff
@@ -0,0 +1,975 @@
+base_images:
+  ansible:
+    name: "4.18"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.18"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  hypershift-operator:
+    name: "4.18"
+    namespace: ocp
+    tag: hypershift-operator
+  openstack-installer:
+    name: "4.18"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.18"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.18"
```

Comment:
4.17？

Reply:
Thanks @JianLi-RH, i will fix it.

Reply:
looks `make update` updated in multiple places wrongly.

Reply:
yes, and please also check other files.

Reply:
fixed